mediapipe/graphs/object_detection/object_detection_desktop_ovms_graph.pbtxt

```diff
@@ -51,6 +51,7 @@ node {
       servable_name: "ssdlite_object_detection"  # servable name inside OVMS
       servable_version: "1"
       server_config: "mediapipe/calculators/ovms/config.json"
+      log_level: "DEBUG"
```

Comment:
Debug will produce a lot of unnecessary logs. Why not info or skip?

Reply:
Will remove used for tests.